src/ui/components/NetworkMonitor/RequestDetails.tsx

```diff
@@ -25,10 +28,28 @@ interface Detail {
   value: string | React.ReactChild;
 }
 
-export const RequestDetailsUnavailable: FC = () => {
+export function RequestDetailsUnavailable({
+  loadedRegions,
+  point,
+}: {
+  loadedRegions: LoadedRegions | null;
+  point: ExecutionPoint;
+}) {
   const dispatch = useDispatch();
   const closePanel = () => dispatch(hideRequestDetails());
 
+  let message;
+  if (loadedRegions && isPointInRegions(loadedRegions.loading, point)) {
+    message = (
+      <>
+        <LoadingProgressBar />
```

Comment:
This progress bar doesn't seem great but in the absence of an indeterminate bar, it seemed like the right one to use here. (I don't think it's worth the effort of us trying to estimate an actual progress value for this use case, but maybe others disagree?)